### PR TITLE
[#326] Publication sync text (connects #326)

### PR DESCRIPTION
**Pull request by razu9861**
screen shot 2017-09-12 at 12 01 55 pm

### CODE REVIEW

**Review by Pr0chin — APPROVED** [review]

**Review by nirojdyola — COMMENTED** [review]
@dogi Is it necessary to show "Not Synced" text after "Sync publication" button ? I think visibility of "Sync publication" button means it is not synced. So I think that label is not necessary to show.

**Comment by lmmrssa** [comment]
@dogi @razu9861 Is this issue to show successfully synced for all previously synced publications? Before it was possible to do resync the same publication. Do we still need that feature? If not then we can just remove **Not Synced** message

**Comment by dogi** [comment]
@razu9861 lets speak about what to do here until the next face to face meeting

**Comment by razu9861** [comment]
@dogi sure

**Comment by lmmrssa** [comment]
@razu9861 leave the initial publication same as it is for now. Once synced the button label can be change to something like 'Re-sync Publication' and color can be changed to Grey or something little dull.

**Comment by razu9861** [comment]
screen shot 2017-09-19 at 5 07 36 pm

File: app/MyApp/app/views/PublicationTable.js [hunk 1]
```diff
@@ -466,7 +467,6 @@
                                                                                                 success: function(response){
                                                                                                     alert(App.languageDict.attributes.Pubs_Replicated_Success)
                                                                                                     App.stopActivityIndicator();
-                                                                                                    that.render()
```
Comment (lmmrssa) [inline app/MyApp/app/views/PublicationTable.js]: why is this code removed

**Review by dogi — APPROVED** [review]

**Review by lmmrssa — APPROVED** [review]

**Pull request closed without merging**